sass/directives/_carousels.scss

```diff
@@ -121,30 +121,32 @@
 
 @mixin resource-continue-block {
```

Comment:
You can add height: 40px, vertically center the content (display: flex, align-items: center) to accomplish what the design requires and then you can remove the top and bottom padding settings. The story asks for 40px height so this way that is decoupled from other variables

This will displace the arrow so absolutely position the arrow to `right: 0`